capstone/src/main/java/servlet/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event{
+    String id; 
+    String location;
+    String summary;
+
+    public void EventSummary(String summary, String id) {
+        this.summary = summary;
+        this.id = id;
+
+    }
+
+     public void Event(String summary, String id, String location) {
+        this.summary = summary;
+        this.id = id;
+        this.location = location;
+
+    }
+    public String getid() {
+        return id;
+    }
+    public String getlocation() {
+        return location;
+    }
+    public String getsummary() {
+        return summary;
+    }
+   
```

Comment:
delete line

Reply:
I'm not clear on which line i should delete. Do you mean this block of code or line 27

Reply:
Line 27